README.md

```diff
@@ -26,13 +26,17 @@ See [blueprint/debian](blueprint/debian) as the canonical example for Debian.
 
 ## Examples
 
-Build a Debian Stretch armhf container called 'debian' (option defaults):
+Create the default container in Docker (specified in `Dockerfile`'s CMD):
```

Comment:
I think we should use just `./build.sh` in the examples and then have a section where we show how you can just substitute `./build-with-docker.sh` if you would like to do the builds in a Docker container. This way, people without docker set up can still build it by installing dependencies manually, or in the case where the user is using my pdsouza/maruos-devbox container as described in the Maru Developer Guide.

Reply:
This is good enough to merge though; I can update the docs in another commit.